public/js/base/gallery.js

```diff
@@ -56,7 +56,7 @@ define(['jquery', 'nunjucks', 'base/ui', 'moment', 'makeapi', 'localized', 'maso
       // Making a custom filter to use it for the client-side l10n
       // Using this filter will help reduce the number of adding
       // variables to the global nunjucks variable.
-      // The usage will be "{{ "some string" | gettext }}"
+      // The usage will be '{{ 'some string' | gettext }}'
```

Comment:
since this describes what you'd use in the HTML, I'm pretty sure this has to stay double quotes. JSCS shouldn't trip over this as it's a comment... I think?